rca/utils/blocks.py

```diff
@@ -7,6 +7,17 @@
 from wagtail.snippets.blocks import SnippetChooserBlock
 
 
+class FeeBlock(blocks.StructBlock):
+    location = blocks.CharBlock(max_length=120, help_text="e.g. Home and EU")
```

Comment:
I'd imagine this can only have two values. This should go back to RCA. I cannot imagine that editors will be disciplined enough to stick to the same values.  We should just use a select box or a separate taxonomy model.

Reply:
I think in the future these may switch to be snippet based, but at the moment they are text fields because in our designs we wave 'overseas', 'overseas students', 'channel islands', etc... so for time sake I'm going to leave this as is